source/features/show-open-prs-of-forks.tsx

```diff
@@ -46,6 +46,13 @@ const countPRs = cache.function(async (forkedRepo: string): Promise<[prCount: nu
 	cacheKey: ([forkedRepo]): string => `prs-on-forked-repo:${forkedRepo}:${getRepo()!.nameWithOwner}`,
 });
 
+// eslint-disable-next-line import/prefer-default-export
+export async function getPRCount(): Promise<number> {
```

Comment:
- Not really a "lint"
- it's only used in `quick-repo-deletion` so it should be there

Reply:
😅Sorry, I kind of treated lint PRs like https://github.com//refined-github/refined-github/issues/5222 again. Will create a separate one for this.